morpheus/_lib/include/morpheus/stages/kafka_source.hpp

```diff
@@ -76,7 +76,7 @@ class KafkaSourceStage : public mrc::pymrc::PythonSource<std::shared_ptr<Message
      * @param async_commits : Asynchronously acknowledge consuming Kafka messages
      */
     KafkaSourceStage(TensorIndex max_batch_size,
-                     std::string topic,
+                     std::vector<std::string> topics,
```

Comment:
Same here, this is a breaking API change, that may not be necessary.

Reply:
Lets overload the constructor here